### PR TITLE
Add video range info to the title

### PR DESCRIPTION
**Pull request by nyanmisaka**
**Changes**
* Add video range info to the title

image

### CODE REVIEW

**Review by crobibero — APPROVED** [review]

File: Emby.Server.Implementations/Localization/Core/en-US.json [hunk 1]
```diff
@@ -9,11 +9,13 @@
     "Channels": "Channels",
     "ChapterNameValue": "Chapter {0}",
     "Collections": "Collections",
+    "Default": "Default",
     "DeviceOfflineWithName": "{0} has disconnected",
     "DeviceOnlineWithName": "{0} is connected",
     "FailedLoginAttemptWithUserName": "Failed login attempt from {0}",
     "Favorites": "Favorites",
     "Folders": "Folders",
+    "Forced": "Forced",
```
Comment (cvium) [inline Emby.Server.Implementations/Localization/Core/en-US.json:18]: Is this related to the range?

Comment (nyanmisaka) [reply Emby.Server.Implementations/Localization/Core/en-US.json:18]: these are some missing translations.

**Review by cvium — APPROVED** [review]
